app-emulation/containerd/files/containerd.service

```diff
@@ -4,7 +4,13 @@ Documentation=http://github.com/docker/containerd
 
 [Service]
 Type=simple
-ExecStart=/usr/bin/containerd --listen unix:///var/run/docker/libcontainerd/docker-containerd.sock --shim /usr/bin/containerd-shim --state-dir /var/run/docker/libcontainerd/containerd --runtime /usr/bin/runc
+ExecStart=/usr/bin/containerd --listen unix:///var/run/docker/libcontainerd/docker-containerd.sock $CONTAINERD_OPTS $CONTAINERD_RUNTIME $CONTAINERD_SHIM $CONTAINERD_START_TIMEOUT $CONTAINERD_STATE_DIR
+
+# define default options that can be overridden
+Environment="CONTAINERD_RUNTIME=--runtime /usr/bin/runc"
```

Comment:
This doesn't matter if the user configures the runtime via `--add-runtime / --default-runtime` on dockerd, right?

I don't see why this knob needs to be turned on containerd ever.